PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -121,6 +122,9 @@ function transformReports(inputReportFiles: string[], index: number)
                 tl.debug("Recognized XUnit report: " + report)
                 reportType = ReportType.XML_XUNIT;
 
+            }  else if (node.name == 'CodingStandards' && !bStaticAnalysisResult) {
```

Comment:
Why do we need "!bStaticAnalysisResult"? Is there any chance that we can set it twice?

Reply:
It turned out "!bStaticAnalysisResult" wasn't needed here because the node 'CodingStandards' would only happen once in our reports. Thx!